src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationMiddleware.cs

```diff
@@ -46,9 +45,14 @@ public class OpenIdConnectAuthenticationMiddleware : AuthenticationMiddleware<Op
         {
             _logger = loggerFactory.CreateLogger<OpenIdConnectAuthenticationMiddleware>();
 
+            if (string.IsNullOrEmpty(Options.SignInScheme))
+            {
+                Options.SignInScheme = externalOptions.Options.SignInScheme;
+            }
```

Comment:
Actually, this specific `SignInScheme` lives on `OpenIdConnectAuthenticationOptions` and is just a wrapper around TokenValidationParameters.AuthenticationType. But of course, it serves the same purpose.

That said, I'm not sure that the passive/active/automatic/manual question is pertinent here, as it can be used in both modes (and anyway, this middleware doesn't inherit from `AutomaticAuthenticationHandler`).